### ISSUE TITLE
Sunday tasks - no need to work today though!

### ISSUE THREAD

**Issue by aaronmberger-nwfsc** [pacific-hake/hake-assessment#516]
Just letting folks know what I'm doing today.  I'm going to run through text from the Title page through the end of the executive summary today.  I'll note where hard coded values (shouldn't be many) will still need to be double checked but the main results in my current document build (with 2199 MCMC samples) will be nearly identical to a document that builds with the 2000.

**Comment by aaronmberger** [comment]
Worked through the executive summary yesterday, still need to do the research and data needs part.  I don't think any changes I made are too different than our standard language.  The exceptions are 1) the following statement in the one page summary:
[The model was updated to include year-specific fecundity from 1975-2018. However, limited
exploratory and supplementary analyses could be completed for the 2019 assessment as
a result of the U.S. government shutdown considerably delaying delivery of U.S. age data
and reducing JTC preparatory time.]

And 2) in the executive summary (data and assessment section):
[The 35 day 2018/2019 U.S. government shutdown considerably delayed delivery of U.S. age data
and reduced the JTC’s collective capacity to complete the 2019 assessment following status-quo
procedures. As a result, many planned supplementary and exploratory analyses could not be completed,
including some developed for response to 2018 SRG requests (see Section 3.3).]

Happy to have edits/comments on this language.  I thought it prudent to make sure the impacts of the shutdown are forefront.
